planner-api/src/main/java/pl/edu/agh/kuce/planner/goal/persistence/SubGoal.java

```diff
@@ -31,20 +31,32 @@ public class SubGoal {
     @Column(nullable = false)
     private Integer amount;
 
+    @Column(nullable = false)
+    private Boolean completed;
+
     public SubGoal() {
```

Comment:
This constructor is not used. Could be deleted

Reply:
Unfortunatelly It can not be deleted :(

Reply:
There are errors that SubGoal does not have default constructor